lib/instrumental.js

```diff
@@ -245,9 +233,19 @@ exports.init = function instrumental_init(startup_time, config, events) {
       recordCounterRates = config.instrumental.recordCounterRates;
     }
 
-    if(typeof(config.instrumental.metricPrefix) === 'undefined'){
-      metricPrefix = "";
-    } else {
+    if(typeof(config.instrumental.metricPrefix) !== 'undefined' &&
+       config.instrumental.metricPrefix != "" &&
```

Comment:
I'm comfortable not.  They'll all get turned into _ downstream, so, there should be no risk of failure, we can let the buyer beware on this one.